code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1198,8 +1198,27 @@
 /datum/reagent/toxin/plasma/plasmavirusfood/weak
 	name = "weakened virus plasma"
 	id = "weakplasmavirusfood"
+	id = "mutates blood"
```

Comment:
You define ID twice. And while it probably doesn't matter other id's don't use spaces so keep it consistent please.